src/components/experiment-tracking/experiment-primary-toolbar/experiment-primary-toolbar.js

```diff
@@ -26,14 +27,13 @@ export const ExperimentPrimaryToolbar = ({
   showRunDetailsModal,
   sidebarVisible,
   setShowRunExportModal,
+  onToggleBookmark,
+  runsMetaData,
```

Comment:
The `d` in `MetaData` should be lowercase.

```suggestion
  runsMetadata,
```

It should be like that in all other places, too.

Reply:
Done.